src/zope/app/publisher/xmlrpc/testing.py

```diff
@@ -87,12 +88,17 @@ def request(self, host, handler, request_body, verbose=0):
 
 
 def ServerProxy(wsgi_app, uri, transport=None, encoding=None,
-                verbose=0, allow_none=0, handleErrors=True):
+                verbose=0, allow_none=0, handleErrors=True,
+                use_datetime=False, use_builtin_types=False):
```

Comment:
```suggestion
                **transport_options):
```

Plus the removal of the new `if six.PY3` branch, which would no longer be necessary.

As written, calling `ServerProxy(use_builtin_types=True)` on Python < 3.5 will do **nothing**, which could lead to very surprising behavior—basically, it's letting an error pass silently. This way, if the server doesn't understand that option you should get a `TypeError`. Yes, that forces considering the implications onto the client, but that seems better than something of a hidden footgun?

Reply:
Implemented as suggested. This allows to drop the dependency on `six`, too.